stable/postgresql/templates/statefulset.yaml

```diff
@@ -104,14 +123,6 @@ spec:
 {{ toYaml .Values.metrics.resources | indent 10 }}
 {{- end }}
       volumes:
```

Comment:
removing this results in the following message from `kubectl get events` even thought the `volumeClaimTemplate` uses the name `data`.

```
7m        8m        16        p1-postgresql.15247964090b421f      StatefulSet                                              Warning   FailedCreate            statefulset-controller                                                             create Pod p1-postgresql-0 in StatefulSet p1-postgresql failed error: Pod "p1-postgresql-0" is invalid: spec.containers[0].volumeMounts[0].name: Not found: "data"
```